.docs/README.md

```diff
@@ -196,45 +181,42 @@ $payment = $service->createPayment([
 ]);
 ```
 
-Zadruhé nastavit adresy, na které Gopay platební brána přesměruje při úspěchu či
-naopak selhání platby.
+Second, set the addresses to which the Gopay payment gateway redirects the success of the payment or while error in payment proccess.
 
 ```php
 $service->setSuccessUrl($this->link('//success', ['orderId' => $orderId]));
 $service->setFailureUrl($this->link('//failure', ['orderId' => $orderId]));
 ```
 
-Je užitečné si poznačit ID platby (například pokud se má platba vázat
-k nějaké objednávce apod.). Toho lze docílit předáním callbacku jako třetího
-parametru metodě `pay()`.
+It is useful to make a note of the payment ID (for example if the payment is to be bound to an order etc.). This can be accomplished by passing the callback as the third parameter to the `pay ()` method.
```

Comment:
It is useful to make a note of the payment ID (for example if the payment is to be bound to an order etc.). This can be accomplished by passing the callback as the third parameter to the `pay()` method.